src/components/UI/SearchBar/index.css

```diff
@@ -3,3 +3,27 @@
   width: 100%;
   position: relative;
 }
+
+.autoComplete {
+  position: absolute;
+  top: 41px;
+  width: 100%;
+  height: 200px;
```

Comment:
maybe we can use max-height here? it looks weird with fea items

![image](https://user-images.githubusercontent.com/188464/173206495-98c6e0ed-a77d-4eca-a718-156bddedfbff.png)

![image](https://user-images.githubusercontent.com/188464/173206504-2b30fca5-1c97-4059-a309-f0b5f083256e.png)

Reply:
done